### DISCUSSION TITLE
About the output file (HDF5)

### DISCUSSION THREAD

**Discussion by Sduwzy** [chombo-discharge/chombo-discharge#471] [Q&A] [answered]
Hello.

Thank you for your job for plasma simulation.

We reran the program using your example and generated the HDF file. However, since we are new to this area, could you please guide us on how to use your HDF5 file? For example, with the following file:

![image](https://github.com/user-attachments/assets/36e34078-1b09-481f-9eb7-170759d7dd0e)
![image](https://github.com/user-attachments/assets/0352ceb3-68ea-4e05-b5cb-b137ee48c668)

**Comment by rmrsk** [comment, marked as answer]
Greetings - we use VisIt for visualizing HDF5 files, see https://chombo-discharge.github.io/chombo-discharge/Base/Control.html#visualization.

**Reply by Sduwzy** [reply]
Thank you for your patient answers, my question has been resolved.
